runtime/v1/shim/client/client.go

```diff
@@ -299,3 +319,14 @@ func (c *Client) waitForExit(pid int) <-chan struct{} {
 	})
 	return c.exitCh
 }
+
+func relCallback(address string, f func(base string) error) error {
+	dir := filepath.Dir(address)
+	base := filepath.Base(address)
+	err := os.Chdir(dir)
```

Comment:
I think `os.Chdir` will impact all the goroutine, especially in concurrent cases. we couldn't do like this.

Reply:
correct, this will apply to the whole process.

Reply:
This is a potential problem even though there no other places using cwd. How about we use a global `cwd` lock to manage cwd. Furthermore, I think forking a thread without CLONE_FS to fix this issue would need more ugly codes